test/unit/junitReporter.js

```diff
@@ -0,0 +1,91 @@
+var junitReportBuilder = require('junit-report-builder');
```

Comment:
I think it would be good to have tests to ensure that the file is being created properly. What happens if the file already exists? We should test that the correct behavior happens for an existing file.